### ISSUE TITLE
recover gracefully without env vars

### ISSUE THREAD

**Issue by jbayer** [cloudfoundry-samples/lattice-app#2]
when running on docker you have to specify env variables or grace doesn't launch well without complaining about the env missing in the logs.

```
docker run --rm -i -t -p 8080:8080 -e INSTANCE_INDEX=0 -e PORT=8080 cloudfoundry/lattice-app /lattice-app
```

would be good to assume port 8080 and have instance index default to "unknown" or "?" when the env vars are not set.

**Comment by davidwadden** [comment]
created https://www.pivotaltracker.com/story/show/90420532 to track this effort.  we will work with our pm to prioritize this into our backlog.

**Comment by davidwadden** [comment]
closing issue -- [#90420532](https://www.pivotaltracker.com/story/show/90420532) has been accepted

this was addressed with 08e31e6a9a580c330ddea48f9d57216aa0f3370c
